dashboard/app/serializers/api/v1/pd/application_serializer.rb

```diff
@@ -10,4 +10,8 @@ def email
   def form_data
     object.full_answers_camelized
   end
+
+  def application_type
+    object.type.split('::').last
+  end
```

Comment:
Note we also have `application_type` (teacher vs. facilitator) and `application_year` attributes